R/Service.R

```diff
@@ -20,13 +20,24 @@ setOldClass("request")
 
 .config <- function(x) x@config
 
+.service_get_api_file <- function(reference_url, reference_headers) {
+    fl <- tempfile()
+    response <- GET(
+        reference_url,
+        add_headers(.headers = reference_headers),
+        write_disk(fl)
+    )
+    .avstop_for_status(response, ".service_validate_md5sum")
```

Comment:
does the 2nd argument accurately reflect where the error is occurring?

Reply:
Thanks! I've resolved this in the latest change